feature_extraction/bag_of_words.py

```diff
@@ -0,0 +1,39 @@
+"""
+Refrences:
+https://en.wikipedia.org/wiki/Bag-of-words_model
+@author QuantumNovice
+
+flake8: ignore
+bag_of_words.py:21:80: E501 line too long (96 > 79 characters)
+bag_of_words.py:24:80: E501 line too long (87 > 79 characters)
```

Comment:
Please do a [doctest normalize whitespace](https://github.com/TheAlgorithms/Python/blob/61fec83242bb00766141ffa2571414bbba68ab6e/maths/prime_factors.py#L21) and then wrap the long lines.

Reply:
On it. Is now resolved in next commit.